anthos-bm-gcp-terraform/docs/nfs.md

```diff
@@ -1,87 +1,64 @@
 ## NFS Shared Storage
 
-An optional NFS server can be provisioned to provide shared storage to the **Anthos on bare metal** cluster.  In the standard [Quick Starter](quickstart.md) method, commands are provided to install the NFS Container Storage Interface (CSI) driver, in the [All in one install](one_click_install.md) or [Manual LB install](manuallb_install.md) methods this is done automatically.
+An optional NFS server can be provisioned to provide shared storage to the **Anthos on bare metal** cluster. If you follow the standard [Quick Starter](quickstart.md) guide, then the commands required to install the **NFS Container Storage Interface (CSI)** driver are provided as part of the Terraform output. If you followed one of [All in one install](one_click_install.md) or [Manual LB install](manuallb_install.md) guides then everything is setup automatically.
```

Comment:
Nit:
`Manual LB` -> `Manual Loadbalancer`

Reply:
I've used the shorten form to be consistent with how we refer to this mode in the [main page of the sample](https://github.com/GoogleCloudPlatform/anthos-samples/tree/nfs-docs/anthos-bm-gcp-terraform#getting-started). Do you think we can stick to that?

Reply:
`LB` sounds good. :)